sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/DocumentProducer.java

```diff
@@ -65,7 +66,10 @@
         void populatePartitionedQueryMetrics() {
             String queryMetricsDelimitedString = pageResult.getResponseHeaders().get(HttpConstants.HttpHeaders.QUERY_METRICS);
             if (!StringUtils.isEmpty(queryMetricsDelimitedString)) {
-                queryMetricsDelimitedString += String.format(";%s=%.2f", QueryMetricsConstants.RequestCharge, pageResult.getRequestCharge());
+                queryMetricsDelimitedString += String.format(Locale.ROOT,
+                                                             ";%s=%.2f",
+                                                             QueryMetricsConstants.RequestCharge,
```

Comment:
Just to double-check my understanding is correct - instead of changing the place where we were Parsing the Double you decided to normalize the locale here because for all other content where we parse floating point numbers the content is created by our backend and we own the locale/formatting, correct?

Reply:
Thats correct Fabian